base/commands/viridian/custom_class_it_test.go

```diff
@@ -0,0 +1,93 @@
+package viridian_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/it"
+)
+
+func customClassTest(t *testing.T) {
+	viridianTester(t, func(ctx context.Context, tcx it.TestContext) {
+		// setup
+		f := "foo.zip"
+		fd := "testdata/" + f
+		cID := ensureClusterRunning(ctx, tcx)
+		// test upload custom class
+		tcx.WithReset(func() {
+			tcx.CLCExecute(ctx, "viridian", "upload-custom-class", cID, fd)
+			tcx.AssertStderrContains("OK")
+			check.Must(waitCustomClassUpload(ctx, tcx))
+			tcx.AssertStderrContains("OK")
```

Comment:
Is this assert necessary? Stderr would already include `OK` from the operation at line 22. Same comment for similar usage.